### PR TITLE
 i18n:translate triage.md to chinese

### PR DESCRIPTION
**Pull request by donhui**

### CODE REVIEW

File: content/contribute/triage.zh.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    parent: "contribute"
+    weight: 30
+weight: 30
+sections_weight: 30
+draft: false
+aliases: [/contribute/triage/]
+toc: true
+---
+
+Jenkins X 项目主要的问题跟踪系统是 https://github.com/jenkins-x/jx/issues。这旨在捕捉问题、想法和开发工作。 如有疑问请提交一个问题，一名 Jenkins X 团队成员将考虑尽快给它分类。
```
Comment (jiangytcn) [inline content/contribute/triage.zh.md]: “如”前的空格 ？

Comment (donhui) [reply content/contribute/triage.zh.md]: thanks
已修正

**Comment by LinuxSuRen** [comment]
/lgtm

Once you reviewed those changes, you can add your comments.

**Comment by donhui** [comment]
> /lgtm
> 
> Once you reviewed those changes, you can add your comments. @jiangytcn
@LinuxSuRen 
ok

**Pull request merged**